src/rct2/S6Importer.cpp

```diff
@@ -479,24 +479,43 @@ extern "C"
 
     int game_load_network(SDL_RWops* rw)
     {
-        bool result = false;
+        bool result = true;
         auto s6Importer = new S6Importer();
-        try
-        {
-            s6Importer->LoadSavedGame(rw);
-            s6Importer->Import();
-
-            openrct2_reset_object_tween_locations();
-            result = true;
-        }
-        catch (ObjectLoadException)
-        {
-            set_load_objects_fail_reason();
-        }
-        catch (Exception)
-        {
-        }
-        delete s6Importer;
+		try {
```

Comment:
C++ files use spaces rather tabs. You accidentally converted the spaces to tabs.